light_curves/code_src/ML_utils.py

```diff
@@ -2,7 +2,7 @@
 from scipy import stats
 from scipy import interpolate
 import matplotlib.pyplot as plt
-import numba
+#import numba
```

Comment:
Generally, do remove lines that are not working (or not used) rather than comment them out. I would think the only exception to this rule of thumb is the code snippets that are around to show how to scale up the same thing for a larger sample, or how to make a temporary data dump, etc.